ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,46 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+The OCF maintains a strict policy of not removing or modifying content
+hosted by user accounts unless legally compelled to do so, or if such
+content is in violation of existing OCF account policies.
+
+This means the OCF cannot, and will not, change or remove any text,
+pictures, information, or other content hosted by an OCF user. Only the
+user who owns the relevant OCF account can modify the data we host on
+their behalf. We do however reserve the right to deactivate inactive
+accounts at our discretion.
+
+### Options for removing content
+
+If you would like a document hosted by the OCF to be modified or removed,
+you have a few options.
+
+1. If the content falls under copyright protection, you may send a DMCA
+takedown notice to the University, which will forward the request to us.
+Do not send any notices to the OCF directly; we can only act on them if
+they are sent to our designated agent.
+
+2. You may try to contact the owner of the account in question and ask them
+to make the relevant modifications. If contact information is not available
+on the site itself, one can send an email to <username>@ocf.berkeley.edu,
+where <username> is the username of the OCF account hosting the content.
+
+3. If repeated attempts to contact the owner of the account fail, you may
+request further contact information for the account from the OCF by
+contacting us at [help@ocf.berkeley.edu](mailto:help@ocf.berkeley.edu).
+We may provide further contact information, if we have any, at our
+discretion.
+
+4. If repeated attempts to contact the owner after step 3 fail, and we
+determine that the account in question is inactive, we may, at our
+discretion, choose to disable the account. However, if the account is
+not inactive and the owner refuses to make the changes, the OCF will not
```

Comment:
Remove this sentence. I'd suggest avoiding modifying user data even with explicit consent because we want to minimize involvement, especially if the action can be ambiguous (rm the file, move the file, chmod the file, change a link, etc.).